deploy-operator/README.md

```diff
@@ -0,0 +1,18 @@
+# deploy-operator
+
+This deploy-operator handlaes the key creation and rotation jobs for each locality we deploy to Kubernetes.
```

Comment:
"handles". Also this README should discuss how the code generation works. What are the inputs -- does the YAML generate the Go, or is it the magic comments in the Go that generate other Go files? When would codegen need to be re-run? How does one run codegen? I'm less concerned with the `deploy` and `install` targets because in any case, those ought to get removed in a subsequent PR, no?

Reply:
I don't plan on removing them in subsequent PRs. Eventually this will be integrated with Helm.

I also don't want to repeat the documentation of kubebuilder in this readme. Do you think a link to kubebuilder would suffice?

Reply:
https://github.com/abetterinternet/prio-server/pull/168/commits/81f7e4546ca3dc3a946513cf994d0d7cbb5c245b Added a quick basic overview of the files